pkg/cli/adbtunnel.go

```diff
@@ -94,11 +94,11 @@ func runOpenADBTunnelCommand(config Config, flags *openADBTunnelFlags, c *comman
 		adbPath = path
 	}
 
-	if config.ADBControlDir == "" {
-		config.ADBControlDir = DefaultADBControlDir
+	if opts.Config.ADBControlDir == "" {
```

Comment:
I was thinking that this logic is better put in config.go, in a step that fills out empty values with these defaults after they are pulled from file.

Reply:
Done